pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -66,19 +66,22 @@ type DefaultPreemption struct {
 
 var _ framework.PostFilterPlugin = &DefaultPreemption{}
 
-type findCandidatesOpts struct {
-	candidateOffset       *int32
-	defaultPreemptionArgs *config.DefaultPreemptionArgs
+type FindCandidatesOptions struct {
```

Comment:
this struct doesn't need to be exported.

Reply:
I completely misunderstood why we want FindCandidates exported haha. The member fields do not need to be exported either; only the `WithOption` functions need to I guess.